app/com/criteo/drelephant/schedulers/LangoustineScheduler.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2016 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.criteo.drelephant.schedulers;
+
+import com.linkedin.drelephant.configurations.scheduler.SchedulerConfigurationData;
+import com.linkedin.drelephant.schedulers.Scheduler;
+import org.apache.log4j.Logger;
+
+import java.util.Properties;
+
+
+/**
+ * This class provides methods to load information specific to the Langoustine scheduler.
+ */
+public class LangoustineScheduler implements Scheduler {
+
+  private static final Logger logger = Logger.getLogger(LangoustineScheduler.class);
+
+  public static final String LANGOUSTINE_JOB_ID = "com.criteo.langoustine.name";
+  public static final String LANGOUSTINE_START_DATE = "com.criteo.langoustine.calc_start";
+  public static final String LANGOUSTINE_PROJECT_NAME = "com.criteo.langoustine.project_name";
+
+  private String schedulerName;
+  private String jobId;
+  private String jobDate;
+  private String jobWorkflow;
+  private String jobName;
+
+  private String jobDefUrl   = ""; // TODO
+  private String jobExecUrl  = ""; // TODO
+  private String flowDefUrl  = ""; // TODO
+  private String flowExecUrl = ""; // TODO
+
+  public LangoustineScheduler(String appId, Properties properties, SchedulerConfigurationData schedulerConfData) {
+
+    String maybeWorkflow = properties.getProperty(LANGOUSTINE_PROJECT_NAME);
+    if (maybeWorkflow == null) {
```

Comment:
or assign jobWorkflow with the properties.getProperty and then check if jobWorkflow is null to decide if you should assign "default_langoustine"